exercises/luhn/test/Tests.hs

```diff
@@ -48,14 +48,22 @@ cases = [ Case { description = "single digit strings can not be valid"
                , input       = "8273 1232 7352 0569"
                , expected    = False
                }
+        , Case { description = "invalid long number with an even remainder"
+               , input       = "1 2345 6789 1234 5678 9012"
+               , expected    = False
+               }
         , Case { description = "valid number with an even number of digits"
                , input       = "095 245 88"
                , expected    = True
                }
+        , Case { description = "valid number with an odd number of spaces"
+               , input       = "234 567 891 234"
+               , expected    = True
+               }
 -- This track is not testing these cases, since we would rather focus on the algorithm,
 -- and because it seems strange to be unable to distinguish between well-formed invalid input and malformed input.
```

Comment:
I'm remarking that the comment above was left by 610ba692834d5e7a7724e7ead0cff7877615010b.